admin/sections/knowledge.tsx

```diff
@@ -1,264 +1,113 @@
-                  <div className="text-sm overflow-hidden">
-                    {file.name.length > 20
-                      ? `${file.name.slice(0, 10)}...${file.name.slice(-10)}`
-                      : file.name}
-                  </div>
-                  <button
-                    className="text-gray-500 text-sm w-2"
-                    onClick={() => handleRemoveFiles([file])}
-                    disabled={isSubmitting}
-                  >
-                    {file.status === "uploaded" || file.status === "failed" ? (
-                      "✖"
-                    ) : (
-                      <LoaderCircle className="animate-spin w-4" />
-                    )}
-                  </button>
-                </div>
-              </div>
-            </TooltipTrigger>
-            <TooltipContent>
-              <p>{file.name}</p>
-            </TooltipContent>
-          </Tooltip>
-        </TooltipProvider>
-      ))}
-    </div>
-  );
-};
-
-const UploadFile = ({
-  processUpload,
-  isSubmitting,
-}: {
-  processUpload: (files: globalThis.File[]) => Promise<void>;
-  isSubmitting: boolean;
-}) => {
-  const inputRef = useRef<HTMLInputElement>(null);
-
-  return (
-    <div className="grid mt-10 w-full max-w-sm items-center gap-1.5">
-      <Input
-        ref={inputRef}
-        id="upload-knowledge-files"
-        type="file"
-        style={{ display: "none" }}
-        multiple
-        onChange={async (e) => {
-          await processUpload(Array.from(e.target.files ?? []));
-          e.target.value = ""; // Clear the input value
-        }}
-      />
-      {!isSubmitting && (
-        <Label htmlFor="upload-knowledge-files">
+      <div className="flex items-center space-x-2">
+        <Switch
+          checked={useLlamaCloud}
+          onCheckedChange={onSwitchKnowledge}
+          id="use-llamacloud"
+        />
+        <Label htmlFor="use-llamacloud">Use LlamaCloud</Label>
+      </div>
+      {useLlamaCloud && (
+        <div className="space-y-4 mt-4">
+          <div className="flex items-center justify-between">
+            <h4 className="font-medium text-lg">LlamaCloud Config</h4>
+            <Button
+              size="sm"
+              variant="secondary"
+              onClick={() => setLlamacloudDialogOpen(true)}
+            >
+              Edit Configuration
+              <Edit className="ml-2 h-4 w-4" />
+            </Button>
+          </div>
+          <LlamaCloudConfigForm form={form} viewOnly />
           <Button
-            onClick={(e) => {
-              e.preventDefault();
-              if (!isSubmitting) {
-                inputRef.current?.click();
-              }
-            }}
+            asChild
+            className="bg-green-500 hover:bg-green-600 mt-2"
+            size="sm"
           >
-            Upload Files
+            <a href="#" target="_blank">
+              Configure Data Sources <ExternalLink className="ml-2 h-4 w-4" />
```

Comment:
@thucpn to simplify this, in this PR, we can also just link to https://cloud.llamaindex.ai